content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
 complexes but may be less effective since on average protein complexes have
 fewer sequence homologs.
 
+### Protein-Protein Interactions
+
+Protein-protein interactions (PPIs) are highly specific and non-accidental physical contacts between proteins which occur for purposes other than generic protein production or degradation [@doi:10.1371/journal.pcbi.1000807].
+PPIs are key to many cellular processes like metabolism and immune responses.
+Abundant interaction data have been generated in-part thanks to advances in high-throughput screening methods, such as yeast two-hybrid and affinity-purification with mass spectrometry.
+However, because many PPIs are transient or dependent on biological context, high-throughput methods can fail to capture a number of interactions.
+Additionally, common types of high-throughput screens for PPIs, such as the yeast two-hybrid, can have issues with high rates of false positive results [@doi:10.1186/s12964-015-0116-8 @doi:10.1002/pmic.200800150].
+The imperfections and costs associated with many experimental PPI screening methods have motivated an interest in high-throughput computational prediction.
+
+This section will focus on advances in *de novo* PPI prediction.
```

Comment:
Perhaps combine this line and the next and connect more directly the the paragraph above.  Something like:

> Machine learning approaches extract PPI from the literature [@doi:10.1016/j.jbi.2007.11.008; @arxiv:1706.01556v2] cannot identify context-specific interactions, motivating *de novo* PPI prediction.

Note that I again added the `;` between references.